docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -4,14 +4,55 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# How to transpile circuits in a pulse-efficient way?"
+    "# Transpile circuits in a pulse-efficient way"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this notebook, we show readers how to transpile circuits in a pulse-efficient way by transpiling the two-qubit gates into the [`RZXGate`](https://qiskit.org/documentation/stubs/qiskit.circuit.library.RZXGate.html). The 'RZXGate' is implemented by turning on the two-qubit $Z\\otimes X$ interaction by applying echoed cross-resonance pulses through the control electronics, hence serving as a continuous gate set native to the hardware. We show examples where this method results in shorter circuit depth and duration. Since qubits have a limited coherence time, a shorter total circuit duration is desirable. The pulse-efficient transpilation has been shown to give more accurate results in the quantum approximate optimization algorithm applied to MAXCUT (see reference [here](https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.3.043088)). We demonstrate how to implement this technique to transpile a relevant circuit in simulating quantum dynamics. And the Runtime result from the hardware suggests that the `RZXGate` type of transpilation produces more accurate results."
+    "*Estimated QPU usage: 14 seconds (tested on IBM Sherbrooke)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Background: What is a pulse gate?\n",
```

Comment:
This background paragraph feels a little out of place as the first thing you read. As a reader I think the first sentence of this document that resonates the best is the first sentence in the "overview" section "This tutorial illustrates how to transpile circuits in a pulse-efficient way by transpiling two-qubit gates into the [`RZXGate`](https://qiskit.org/documentation/stubs/qiskit.circuit.library.RZXGate.html)". I think it might be better to combine the background and overview section into a singular section, starting with the first sentence explaining what will be covered in the notebook